src/main/java/io/vertx/ext/stomp/StompServerHandler.java

```diff
@@ -0,0 +1,321 @@
+package io.vertx.ext.stomp;
+
+import io.vertx.codegen.annotations.Fluent;
+import io.vertx.codegen.annotations.VertxGen;
+import io.vertx.core.AsyncResult;
+import io.vertx.core.Handler;
+import io.vertx.core.Vertx;
+import io.vertx.ext.stomp.impl.DefaultStompHandler;
+
+import java.util.List;
+
+/**
+ * STOMP server handler implements the behavior of the STOMP server when a specific event occurs. For instance, if
+ * let customize the behavior when specific STOMP frames arrives or when a connection is closed. This class has been
+ * designed to let you customize the server behavior. The default implementation is compliant with the STOMP
+ * specification. In this default implementation, not acknowledge frames are dropped.
+ */
+@VertxGen
+public interface StompServerHandler extends ServerFrameHandler {
```

Comment:
If this is public so the user can provide customised implementations then it should be sufficient to just implement the frame handling methods - not sure why we need to expose transaction and subscription stuff - these seem to me to be implementation details.